src/operations/aur_install/aur_fetch.rs

```diff
@@ -39,6 +39,7 @@ impl AurFetch {
         pb.finish_with_message("All packages found".green().to_string());
         normal_output!();
 
+        print!("  "); // TODO: Make this less hacky
```

Comment:
uh uhm. What's the issue?

Reply:
When displaying AUR packages, the first element of the list isn't padded, however all subsequent elements are

Reply:
This is also the only instance where this seems to happen, potentially indicative of an issue introduced in my `padding` addition to wrap_text()? I'm not sure

Reply:
Isn't padding configured to only pad subsequent entries?

Reply:
Did we add indentation to lists before your implementation?

Reply:
Otherwise maybe print_list needs to be changed

Reply:
Figured this out, pushing change